forte/command_line/generate_ontology/__main__.py

```diff
@@ -194,6 +194,18 @@ def main():
         help="If True, will not enforce prefix check.",
     )
 
+    create_parser.add_argument(
```

Comment:
Can we try to incorporate this new feature into the documentation here? https://github.com/asyml/forte/blob/master/docs/toc/ontology_generation.md

Reply:
probably still no updates to the readme?